boilerplate/fullstack/next-app-dir/app/components/sessionAuthForNextJS.tsx

```diff
@@ -7,9 +7,5 @@ type Props = Parameters<typeof SessionAuth>[0] & {
 };
 
```

Comment:
We don't need this file anymore, and instead, we should see where we use SessionAuthForNextJS and try and use SessionAuth directly in there.

Keep in mind that in home.tsx, we use SessionAuth around the main html cause we want to auto handle the 401 redirection on the frontend.

Reply:
This file is still necessary because it adds the `use client` directive to `SessionAuth`.
Maybe we could remove this (and `SessionAuth`) from `home.tsx` and provide a workaround, but I don't think we should:
- we should show `SessionAuth` somewhere in the example
- SSR seems to work fine as-is (I can see the children of SessionAuth pre-rendered in the response)

Reply:
Oh! Really. That's surprising. I thought SessionAuth will render `null` cause that's what i saw during the demo when this example app was created, and also that's what we had discussed early on. Maybe im wrong - let's discuss this on call.